commands/initial/templates/examples/example.component.html

```diff
@@ -0,0 +1 @@
+<div class="welcome">Welcome</div>
```

Comment:
What does this HTML page do? Does it act as the `index.html`-type page where the tests are run?